chain/polkadot/defaults.go

```diff
@@ -21,6 +21,8 @@ var (
 	DefaultConfig = string("./chain/polkadot/config.toml")
 	// DefaultBasePath Default node base directory path
 	DefaultBasePath = string("~/.gossamer/polkadot")
+	// DefaultMetricsAddress is the default metrics server listening address.
+	DefaultMetricsAddress = "localhost:9876"
```

Comment:
why is this the default now?

Reply:
It seems that it is what we were using before https://github.com/ChainSafe/gossamer/blob/64425445e6be472ae1e2dffba85f8ce4221d2eb8/chain/gssmr/defaults.go#L28